### PR TITLE
added fixed positioning for hangover dropdowns; added scrollParentCla…

### PR DESCRIPTION
**Pull request by corescan**
…ss prop for onScroll listening

### CODE REVIEW

File: src/Select.js [hunk 1]
```diff
@@ -54,6 +89,45 @@ export default class Select extends React.Component {
...
+    let source = parent;
+    let offsetTop = 0;
+    let scrollParentTop = this.scrollParent.scrollTop;
+    while (source) {
+      offsetTop += source.offsetTop;
+      source = source.offsetParent;
+    }
+    return {
+      fixedDropdownStyle: {
+        width: `${parent.offsetWidth}px`,
```
Comment (isuttell) [inline src/Select.js]: Don't technically need to include `px` react should automatically handle it

File: src/Select.js [hunk 2]
```diff
@@ -54,6 +89,45 @@ export default class Select extends React.Component {
...
+        position: 'fixed',
+        top: `${(offsetTop - scrollParentTop) + parent.offsetHeight}px`
+      }
+    };
+  }
+
+  handleScrollParent() {
+    let state = Object.assign({}, this.getDropdownStyle(), {active: false});
```
Comment (isuttell) [inline src/Select.js]: Do we need the update the position if we're hiding it?

Comment (corescan) [inline src/Select.js]: No we don't, I think we could put that update inside toggleActive() when active is true

File: src/Select.js (hunk 1 above)
Comment (corescan) [inline src/Select.js]: cool

**Pull request merged**